src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -64,14 +64,14 @@ const MAX_POLL_ATTEMPTS = 20
 const queryForBidPosition = (bidderPositionID: string) => {
   return metaphysics({
     query: `
-      {
+      query ConfirmBidBidderPositionQuery($bidderPositionID: String!) {
```

Comment:
out of curiosity, can you provide some more context for this change? Why name the query?

Reply:
This used to be a dynamic query due to the presence of the string interpolation, which halts static analysis done by GraphQL. I've changed it to use `variables` to inject `bidderPositionID`. The query name could be any name, but I followed Relayjs' convention of `ComponentNameQueryName(Query|Mutation)`.